src/marines/my_console.py

```diff
@@ -0,0 +1,18 @@
+def function1():
```

Comment:
what does function mean?

Reply:
I use the name provided in the slide.
Renamed by load_integer_array